mac/GUAM.MAC

```diff
@@ -114,6 +115,11 @@
         LA     15,=A(&SYSLIST(5))
         ST     15,8(1)
 .SKPLIT2 ANOP
+        AIF    ('&SYSLIST(6)'(1,1) LT '0' OR '&SYSLIST(6)'(1,1) GT '8')X
+               .SKPLIT3                      #530
+        LA     15,=A(&SYSLIST(6))            #530
+        ST     15,12(1)                      #530
+.SKPLIT3 ANOP  ,                             #530
```

Comment:
Just an explanation of what this code does. The portion of GUAM.MAC is processing a "GUAM WINDOW,VIEW,[SCREEN or GRAPH',a,b,c" request. The existing code properly handles the "a" and "b" operands but not the "c" operand. The code added after .SKIPLKIT1 handles the "c" operand. In short, the update is correct.